docs/examples/draft/galaxy-collection-wf-crate/ro-crate-metadata.json

```diff
@@ -0,0 +1,397 @@
+                },
+                {
+                    "@id": "#advanced-param"
+                },
+                {
+                    "@id": "urn:uuid:eabc423a-227e-4096-8e14-74d0088c8ef9"
+                }
+            ],
+            "name": "collection_workflow",
+            "output": [
+                {
+                    "@id": "urn:uuid:eabc423a-227e-4096-8e14-74d0088c8ef9"
+                },
+                {
+                    "@id": "urn:uuid:4a0f4078-5aff-4e02-9f9c-4ad510050e54"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#galaxy"
+            }
+        },
+        {
+            "@id": "#b484db83-c270-47ab-a7b9-50723cd9212a",
+            "@type": "SoftwareApplication",
+            "name": "Galaxy workflow engine"
+        },
+        {
+            "@id": "#e671bc15-59f0-44cc-bbe2-bf4fcc7e52d3",
+            "@type": "OrganizeAction",
+            "instrument": {
+                "@id": "#b484db83-c270-47ab-a7b9-50723cd9212a"
+            },
+            "name": "Run of Galaxy workflow engine",
+            "startTime": "2022-12-12T17:54:18.224230"
+        },
+        {
+            "@id": "#lineNum-pv",
+            "@type": "PropertyValue",
+            "exampleOfWork": {
+                "@id": "#lineNum-param"
+            },
+            "name": "lineNum",
+            "value": "2"
+        },
+        {
+            "@id": "#lineNum-param",
+            "@type": "FormalParameter",
+            "additionalType": "None",
+            "description": "select 3 lines",
+            "name": "select lines parameter",
+            "valueRequired": true
+        },
+        {
+            "@id": "urn:uuid:d4e573fa-227f-4cdc-94a0-ac959de105e7",
+            "@type": "FormalParameter",
+            "additionalType": "File",
+            "description": "uploaded txt file",
+            "name": "input13"
+        },
+        {
+            "@id": "urn:uuid:ba8af7ec-ef40-405a-bcc4-33c2df9fffed",
+            "@type": "FormalParameter",
+            "additionalType": "File",
+            "description": "uploaded txt file",
+            "name": "input14"
+        },
+        {
+            "@id": "urn:uuid:369edbf0-e16a-45d8-bad8-b0ecb4f566a2",
+            "@type": "FormalParameter",
+            "additionalType": "File",
+            "description": "uploaded txt file",
+            "name": "input12"
+        },
+        {
+            "@id": "#advanced-pv",
+            "@type": "PropertyValue",
+            "exampleOfWork": {
+                "@id": "#advanced-param"
+            },
+            "name": "advanced",
+            "value": "{\"conflict\": {\"__current_case__\": 0, \"duplicate_options\": \"suffix_conflict\", \"suffix_pattern\": \"_#\"}}"
+        },
+        {
+            "@id": "#advanced-param",
+            "@type": "FormalParameter",
+            "additionalType": "None",
```

Comment:
This should probably be "Text" (looking at the value in the corresponding pv). If it's not straightforward to convert from the Galaxy type, better not add the "additionalType" property at all ("None" is not in the RO-Crate context).